src/main/java/com/capgemini/molveno/controller/api/OrderController.java

```diff
@@ -0,0 +1,47 @@
+package com.capgemini.molveno.controller.api;
+
+import com.capgemini.molveno.model.Order;
+import com.capgemini.molveno.repository.OrderRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/api/order")
+public class OrderController {
+
+    @Autowired
+    private OrderRepository orderRepository;
+
+    @GetMapping
+    public Collection<Order> get() {
+        return this.orderRepository.findAll();
+    }
+
+    @GetMapping("{id}")
```

Comment:
Doe dit bij de mappings, dit zorgt ervoor dat we later geen issues gaan krijgen omdat het een string is ipv een json object.

```
produces = MediaType.APPLICATION_JSON_VALUE
```